src/weekly/query.js

```diff
@@ -118,8 +117,7 @@ async function queryZuora (deliveryDate, config: Config) {
         SoldToContact.PostalCode,
         SoldToContact.State,
         Subscription.CanadaHandDelivery__c,
-        Account.MRR,
-        Account.Currency
+        MRR
```

Comment:
Should we be getting the TransactionCurrency here as well?

Reply:
Now explained better how the result set works above.